crazyflie-link/examples/benchmark.rs

```diff
@@ -44,22 +44,24 @@ fn main() -> anyhow::Result<()> {
     purge_crazyflie_queues(&link);
     let mut ping_times = Vec::new();
     for i in 0..opt.n_packets {
-        let mut packet = vec![0; opt.size_packet];
-        packet[0] = 0xf0;
-        packet[1] = i as u8;
+        let mut packet = Packet::new_from_header(0xf0);
+        packet.append_data(&mut vec![i as u8]);
         let ping_time = bench(|| {
+            let data;
+
             link.send_packet(packet)?;
             loop {
                 packet = link.recv_packet_timeout(std::time::Duration::from_secs(10))?;
-                if packet[0] == 0xf0 {
+                if packet.get_header() == 0xf0 {
```

Comment:
Likely broken since bits 3 and 4 cannot be 0 anymore (see bellow).

Should likely be replaced by a test on port/channel anyway?